tests/app/main/helpers/test_sum_counts.py

```diff
@@ -1,135 +1,102 @@
+            }
+        }, created_at='today') == {'created_at': 'today', 'one': 0, 'two': 0, 'three': 0}
 
 
 class TestSumCounts(TestCase):
     def test_summing_without_filtering(self):
-        assert_equal(
-            _sum_counts([
-                {'count': 1},
-                {'count': 2},
-                {'count': 3},
-                {'count': 4}
-            ]),
-            10
-        )
+        assert _sum_counts([
+            {'count': 1},
+            {'count': 2},
+            {'count': 3},
+            {'count': 4}
+        ]) == 10
 
     def test_summing_filtering_on_one_attribute(self):
-        assert_equal(
-            _sum_counts([
+        assert _sum_counts(
+            [
                 {'count': 1, 'include': False},
                 {'count': 2, 'include': True},
                 {'count': 3, 'include': True},
                 {'count': 4, 'include': False}
-            ], {
-                'include': True
-            }),
-            5
-        )
+            ],
+            {'include': True},
+        ) == 5
 
     def test_summing_filtering_on_multiple_attributes(self):
-        assert_equal(
-            _sum_counts([
+        assert _sum_counts(
+            [
                 {'count': 1, 'include': False, 'exclude': True},
                 {'count': 2, 'include': True, 'exclude': False},
                 {'count': 3, 'include': True, 'exclude': True},
                 {'count': 4, 'include': False, 'exclude': False}
-            ], {
+            ],
+            {
                 'include': True,
-                'exclude': False
-            }),
-            2
-        )
+                'exclude': False,
+            },
+        ) == 2
 
     def test_summing_filtering_on_various_acceptable_attributes(self):
-        assert_equal(
-            _sum_counts([
+        assert _sum_counts(
+            [
                 {'count': 1, 'colour': 'cyan'},
                 {'count': 2, 'colour': 'yellow'},
                 {'count': 3, 'colour': 'magenta'},
                 {'count': 4, 'colour': 'black'}
-            ], {
-                'colour': ['cyan', 'yellow', 'magenta']
-            }),
-            6
-        )
+            ],
+            {'colour': ['cyan', 'yellow', 'magenta']},
+        ) == 6
 
     def test_summing_by_different_column(self):
-        assert_equal(
-            _sum_counts([
+        assert _sum_counts(
+            [
                 {'count': 1, 'new_count': 10},
                 {'count': 2, 'new_count': 20},
                 {'count': 3, 'new_count': 30},
                 {'count': 4, 'new_count': 40}
-            ], sum_by='new_count'),
-            100
-        )
+            ],
+            sum_by='new_count',
+        ) == 100
 
 
 class TestFind(TestCase):
     def test_find_string(self):
-        assert_true(
-            _find('word', 'word'),
-        )
-        assert_false(
-            _find('drow', 'word')
-        )
+        assert _find('word', 'word')
```

Comment:
Might it be worth asserting `is True` and `is False` for all these `assert _find()` tests?  Probably fine as is though.